### DISCUSSION TITLE
Where's the option to change the layout of the floating window in "outline"?

### DISCUSSION THREAD

**Discussion by idelice** [nvimdev/lspsaga.nvim#1290] [Ideas]
This doesn't look nice.

I would like to center this to my window with paddings on each corner so it looks good and i can see exactly what code I have in that function without navigating to it.

Or, at least give me an option to modify it because right now, it triggers my OCD.

<img width="476" alt="Screenshot 2023-09-24 at 00 11 41" src="https://github.com/nvimdev/lspsaga.nvim/assets/59118341/3f632113-5832-4c6b-80c8-b010189c23b7">

**Comment by glepnir** [comment]
currently you can change the size of float window . no other layout for outline preview.

**Reply by idelice** [reply]
Yes but how ??

**Reply by glepnir** [reply]
what's the commit of your lspsaga ..default preview width should be 0.7percent or max content width. and there also have `outline.layout = 'float'`

**Reply by idelice** [reply]
I tried all configs for the layout and it made no change to the floating preview window.

**Comment by glepnir** [comment]
or if still think it not works fine for you create an imporve issue. maybe we provide a float preview width or some layout for preview width.
